dropshot/src/test_util.rs

```diff
@@ -451,6 +451,12 @@ impl LogContext {
         &self.test_name
     }
 
+    /// Returns the log path.
+    #[inline]
```

Comment:
```suggestion
    #[inline]
```

Can we make this non-`pub`?

Reply:
I added the method so it could be used from within the test module to read the log file and check if the entry was there, so it had to be `pub` in order for it to be accessible (I think?). That being said, since the check is no longer there, I've removed this bit too.